<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1154</width>
    <height>772</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Logo Manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../lmgr.qrc">
    <normaloff>:/ico/ico/app.ico</normaloff>:/ico/ico/app.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_0">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Logo Screens</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_1" stretch="0,1,0">
         <item>
          <widget class="QLabel" name="label_1_txt">
           <property name="toolTip">
            <string>Size, Offset, Length</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="text">
            <string notr="true">
?
</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_1_img">
           <property name="minimumSize">
            <size>
             <width>216</width>
             <height>468</height>
            </size>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_1">
           <item>
            <widget class="QPushButton" name="pushButton_1_imp">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Import Logo Screen</string>
             </property>
             <property name="text">
              <string>Import</string>
             </property>
             <property name="icon">
              <iconset resource="../lmgr.qrc">
               <normaloff>:/png/png/import.png</normaloff>:/png/png/import.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_1_exp">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Export Logo Screen</string>
             </property>
             <property name="text">
              <string>Export</string>
             </property>
             <property name="icon">
              <iconset resource="../lmgr.qrc">
               <normaloff>:/png/png/export.png</normaloff>:/png/png/export.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,0">
         <item>
          <widget class="QLabel" name="label_2_txt">
           <property name="toolTip">
            <string>Size, Offset, Length</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="text">
            <string notr="true">
?
</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2_img">
           <property name="minimumSize">
            <size>
             <width>216</width>
             <height>468</height>
            </size>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="pushButton_2_imp">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Import Logo Screen</string>
             </property>
             <property name="text">
              <string>Import</string>
             </property>
             <property name="icon">
              <iconset resource="../lmgr.qrc">
               <normaloff>:/png/png/import.png</normaloff>:/png/png/import.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_2_exp">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Export Logo Screen</string>
             </property>
             <property name="text">
              <string>Export</string>
             </property>
             <property name="icon">
              <iconset resource="../lmgr.qrc">
               <normaloff>:/png/png/export.png</normaloff>:/png/png/export.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1,0">
         <item>
          <widget class="QLabel" name="label_3_txt">
           <property name="toolTip">
            <string>Size, Offset, Length</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="text">
            <string notr="true">
?
</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3_img">
           <property name="minimumSize">
            <size>
             <width>216</width>
             <height>468</height>
            </size>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="pushButton_3_imp">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Import Logo Screen</string>
             </property>
             <property name="text">
              <string>Import</string>
             </property>
             <property name="icon">
              <iconset resource="../lmgr.qrc">
               <normaloff>:/png/png/import.png</normaloff>:/png/png/import.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_3_exp">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Export Logo Screen</string>
             </property>
             <property name="text">
              <string>Export</string>
             </property>
             <property name="icon">
              <iconset resource="../lmgr.qrc">
               <normaloff>:/png/png/export.png</normaloff>:/png/png/export.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,1,0">
         <item>
          <widget class="QLabel" name="label_4_txt">
           <property name="toolTip">
            <string>Size, Offset, Length</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="text">
            <string notr="true">
?
</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4_img">
           <property name="minimumSize">
            <size>
             <width>216</width>
             <height>468</height>
            </size>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPushButton" name="pushButton_4_imp">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Import Logo Screen</string>
             </property>
             <property name="text">
              <string>Import</string>
             </property>
             <property name="icon">
              <iconset resource="../lmgr.qrc">
               <normaloff>:/png/png/import.png</normaloff>:/png/png/import.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_4_exp">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Export Logo Screen</string>
             </property>
             <property name="text">
              <string>Export</string>
             </property>
             <property name="icon">
              <iconset resource="../lmgr.qrc">
               <normaloff>:/png/png/export.png</normaloff>:/png/png/export.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,1,0">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_5_txt">
            <property name="toolTip">
             <string>Size, Offset, Length</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="text">
             <string notr="true">
?
</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5_img">
            <property name="minimumSize">
             <size>
              <width>216</width>
              <height>468</height>
             </size>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="pushButton_5_imp">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Import Logo Screen</string>
              </property>
              <property name="text">
               <string>Import</string>
              </property>
              <property name="icon">
               <iconset resource="../lmgr.qrc">
                <normaloff>:/png/png/import.png</normaloff>:/png/png/import.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_5_exp">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Export Logo Screen</string>
              </property>
              <property name="text">
               <string>Export</string>
              </property>
              <property name="icon">
               <iconset resource="../lmgr.qrc">
                <normaloff>:/png/png/export.png</normaloff>:/png/png/export.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1154</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionFlash"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string notr="true">toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionExit"/>
   <addaction name="separator"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionFlash"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
   <addaction name="actionHelp"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../lmgr.qrc">
     <normaloff>:/png/png/open.png</normaloff>:/png/png/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open Logo Image</string>
   </property>
   <property name="statusTip">
    <string>Open Logo Image</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../lmgr.qrc">
     <normaloff>:/png/png/save.png</normaloff>:/png/png/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save Logo Image</string>
   </property>
   <property name="statusTip">
    <string>Save Logo Image</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../lmgr.qrc">
     <normaloff>:/png/png/exit.png</normaloff>:/png/png/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit Program</string>
   </property>
   <property name="statusTip">
    <string>Exit Program</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../lmgr.qrc">
     <normaloff>:/png/png/info.png</normaloff>:/png/png/info.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About Program</string>
   </property>
   <property name="statusTip">
    <string>About Program</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="../lmgr.qrc">
     <normaloff>:/png/png/help.png</normaloff>:/png/png/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="toolTip">
    <string>Show Help</string>
   </property>
   <property name="statusTip">
    <string>Show Help</string>
   </property>
  </action>
  <action name="actionFlash">
   <property name="icon">
    <iconset resource="../lmgr.qrc">
     <normaloff>:/png/png/flash.png</normaloff>:/png/png/flash.png</iconset>
   </property>
   <property name="text">
    <string>Flash</string>
   </property>
   <property name="toolTip">
    <string>Flash Logo Image</string>
   </property>
   <property name="statusTip">
    <string>Flash Logo Image</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../lmgr.qrc"/>
 </resources>
 <connections/>
</ui>
